README.md

```diff
@@ -4,13 +4,13 @@ Marl is a hybrid thread / fiber task scheduler written in C++ 11.
 
 ## About
 
-Marl is a C++ 11 library that provides a fluent interface for running tasks across a number of threads.
+Marl is a C++ 11 library that provides a fluent interface for running tasks across saveral threads.
```

Comment:
I assume you mean several?

I think `a number of` is more correct - the number is configurable - scaling from 1 to many. An even more correct suggestion might be: `across a configurable number of threads`